Take default og:site_name from sphinx `project` config value (#83)
wpilibsuite/sphinxext-opengraph: README.md

* `ogp_description_length`
    * Configure the amount of characters taken from a page. The default of 200 is probably good for most people. If something other than a number is used, it defaults back to 200. 
* `ogp_site_name`

>     * This is not required. Name of the site. This is displayed above the title.  Taken from the Sphinx `project` config value if not given.

Reviewer: Wording suggestion:
```suggestion
    * This is not required. Name of the site. This is displayed above the title. Defaults to the Sphinx `project` config value.
```
